libmscore/layout.cpp

```diff
@@ -296,9 +296,9 @@ void Score::layoutChords1(Segment* segment, int staffIdx)
                               else
                                     break;
                               }
-                        for (int i = int(downStemNotes.size()) - 1; i >= 0; --i) {
-                              if (downStemNotes[i]->line() <= bottomUpNote->line() + 1)
-                                    overlapNotes.append(downStemNotes[i]);
+                        for (size_t i = downStemNotes.size(); i > 0; --i) { // loop most probably needs to be in this reverse order
```

Comment:
There are three types of comments in this PR: "most probably", "probably" and just "...needs to be...". Why? :)

Reply:
Same here, `i >= 0` is a common situation for inverted loops. `i >= 0` is an inverted `i < 0` expression, so there is nothing wrong. Could you please remove `-1` from the `operator[]` calls and revert back the "-1" in the index initialisation?

Reply:
a) because for 2 I'm sure it is needed (mtests) fail) for the others I've just been told that thes may leed to triouble when inverted
b) that'd lead to other compiler warnings. If `i` is `unsigned` (or `size_t`) the comparision of it being greater or equal to 0 ie meaningless, there is no way for an unsigned value to becoms smaller than 0